price-feeder/oracle/provider/huobi.go

```diff
@@ -31,19 +31,21 @@ type (
 	// API.
 	//
 	// REF: https://huobiapi.github.io/docs/spot/v1/en/#market-ticker
+	// REF : https://huobiapi.github.io/docs/spot/v1/en/#get-klines-candles
 	HuobiProvider struct {
 		wsURL           url.URL
 		wsClient        *websocket.Conn
 		logger          zerolog.Logger
 		mu              sync.Mutex
 		tickers         map[string]HuobiTicker // market.$symbol.ticker => HuobiTicker
+		candles         map[string]HuobiCandle // market.$symbol.kline.$period => HuobiCandle
 		subscribedPairs []types.CurrencyPair
 	}
 
 	// HuobiTicker defines the response type for the channel and
 	// the tick object for a given ticker/symbol.
 	HuobiTicker struct {
-		CH   string    `json:"ch"` // Data belonged channel，Format：market.$symbol.ticker
+		CH   string    `json:"ch"` // Channel name.$symbol.ticker
```

Comment:
[Docs](https://huobiapi.github.io/docs/spot/v1/en/#market-ticker) seem to suggest `market.$symbol.ticker`
Is that correct?

Reply:
it is `market.$symbol.ticker`, fixed that bit